lib/ipmi_backend_utils.pm

```diff
@@ -83,14 +83,10 @@ sub setup_console_in_grub {
             $bootmethod = "module";
             $search_pattern = "vmlinuz";
 
-            my $dom0_options = "";
-            if (get_var("ENABLE_SRIOV_NETWORK_CARD_PCI_PASSTHROUGH")) {
-                $dom0_options = "iommu=on";
-            }
             $cmd
               = "sed -ri '/multiboot/ "
               . "{s/(console|loglevel|loglvl|guest_loglvl)=[^ ]*//g; "
-              . "/multiboot/ s/\$/ $dom0_options $com_settings loglvl=all guest_loglvl=all sync_console/;}; "
```

Comment:
Why remove ```$dom0_options``` ? I think you need to keep this variable at least.

Reply:
the variable was added for this parameter. As no parameter is kept for dom0, this variable is useless. If there would be any new parameters, a new variable or other ways can be used freely.